sycl/doc/extensions/experimental/sycl_ext_oneapi_device_global.asciidoc

```diff
@@ -1124,7 +1196,7 @@ application submits the same kernel from more than one kernel bundle.
 
 == Non-normative: Future anticipated changes, not enabled within this version of the specification
 
-In a future version of this extension, it is expected that when {cpp}20 support is available and enabled, the `consteval` keyword will be used to enable compile-time constant initialization of the device allocations backing `device_global`.  This will simplify some coding patterns, compared with the current zero-initialization requirement.
+In a future version of this extension, it is expected that when {cpp}20 support is available and enabled, the `consteval` keyword will be used to enable compile-time constant initialization of the device allocations backing `device_global` without the `device_image_scope` property.  This will simplify some coding patterns, compared with the current zero-initialization requirement.
```

Comment:
This sentence doesn't make any sense with your new restrictions.  C++20 support is available; that's why we're supporting consteval on device_image_scope device_globals.